tests/spaces/test_so.py

```diff
@@ -0,0 +1,112 @@
+import jax.numpy as jnp
```

Comment:
As mentioned yesterday, we can use this code: https://github.com/NoemieJaquier/MaternGaBO/blob/main/BoManifolds/kernel_utils/kernels_so.py as "unofficial" test, if needed.